packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -9,6 +9,7 @@ const fn = new lambda.Function(this, 'MyFunction', {
     runtime: lambda.Runtime.NodeJS810,
     handler: 'index.handler'
     code: lambda.Code.inline('exports.handler = function(event, ctx, cb) { return cb(null, "hi"); }'),
+    deadLetterQueueEnabled: true
```

Comment:
I wouldn't add this in the first example. You can add a section that describes how to define a DLQ if you want (not required)

Reply:
done